PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Utils/Views/OptionTableViewCell.m

```diff
@@ -43,8 +43,13 @@ - (void)setOption:(PlayerSelectionOption *)option {
   
   [self updateWithState:@([[AssetPersistenceManager sharedManager] downloadStateForEmbedCode:_option.embedCode]) title:_option.title];
   
-  [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleAssetStateChanged:) name:AssetPersistenceStateChangedNotification object:nil];
-  [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleProgressChanged:) name:AssetDownloadProgressNotification object:nil];
+  [[NSNotificationCenter defaultCenter] addObserver:self
+                                           selector:@selector(handleAssetStateChanged:) name:AssetPersistenceStateChangedNotification
```

Comment:
To keep consistency, let's move name to a new line.

Reply:
I think you missed this one.

Reply:
![snip20181001_1](https://user-images.githubusercontent.com/7861904/46317573-15ba5000-c599-11e8-8ed4-921c4e582df9.png)

It's already solved. I don't know why github doesn't show the changes.